ldap/servers/slapd/back-ldbm/db-mdb/mdb_import.c

```diff
@@ -668,6 +668,43 @@ dbmdb_import_run_pass(ImportJob *job, int *status)
 
 /* Helper function to make up filenames */
 
+/* Ensure that the task status and exit code is properly set */
+static void
+dbmdb_task_finish(ImportJob *job, int ret)
+{
+    ldbm_instance *inst = job->inst;
+    char *opname = "importing";
+    char *task_dn = "";
```

Comment:
Shouldn't it be initialized with NULL ?. Later it is freed and in case it is not malloc somewhere it will free a stack address

Reply:
Cannot use NULL because it is used in a log message.
And the free is done in `if  (*task_dn) ` block witch is false if task_dn is ""